50.md

```diff
@@ -26,6 +26,9 @@ Relays SHOULD interpret the query to the best of their ability and return events
 Relays SHOULD perform matching against `content` event field, and MAY perform
 matching against other fields if that makes sense in the context of a specific kind. 
 
+Results SHOULD be returned in the order of their matching score, not by the usual `.created_at`. 
+The `limit` filter SHOULD be applied after sorting by matching score.
+
```

Comment:
```suggestion
```suggestion
Results SHOULD be returned in descending order by quality of search result (as defined by the implementation),
not by the usual `.created_at`. The `limit` filter SHOULD be applied after sorting by matching score.

```